client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -42,6 +44,11 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 
 	getChangeHandler = field => event => {
 		this.setState( { [ field ]: event.target.value } );
+
+		if ( ! this.changed[ field ] ) {
+			this.props.recordJpoEvent( `calypso_jpo_${ field }_changed` );
```

Comment:
Programmatically generated event name 🙀

Reply:
Good catch, fixed in https://github.com/Automattic/wp-calypso/pull/22385/commits/6f79f7acf5a441088b07e59193ad51041970f736